Application/src/visuals/ActorClass.java

```diff
@@ -0,0 +1,144 @@
+package visuals;
+
+import javafx.scene.canvas.GraphicsContext;
+import javafx.scene.image.Image;
+import javafx.scene.paint.Color;
+
+/**
+ * An ActorClass wrapper modeled after DiagramClass.
+ * @author Pontus Laestadius
+ * collaborator Kosara Golemshinska
+ * @version 1.0
+ */
+public class ActorClass implements Renderable {
+
+	private Coordinates coordinates; // The coordinates of the object.
```

Comment:
Is the indentation weird here or is it just git?

Reply:
Looks fine for me, I'll fix it and push again